src/plugins/vis_type_xy/public/types/constants.ts

```diff
@@ -6,52 +6,43 @@
  * Side Public License, v 1.
  */
 
-import { $Values } from '@kbn/utility-types';
-
-export const ChartMode = Object.freeze({
```

Comment:
@nickofthyme is like an enum. What do you think about replacing this code with a clearer definition?

Reply:
I think it's fine to change it back I just don't think babel does a great job transpiling them rn. See https://github.com/elastic/kibana/pull/53536#discussion_r363792364